tests/plt/plot_test.py

```diff
@@ -0,0 +1,189 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+# @file
+
+import scipp as sc
+from scipp.plt import Plot, Model, Figure, widgets, Node
```

Comment:
Isn't the whole point of MVC that we can mock or fake components? So we should not use `Figure` here, otherwise we are back to the old tests that only check if there are any exceptions.

So I think we should test all components in isolation and only have a few tests that make sure that they fit together.

Reply:
Yes, see comments above. I don't intend to keep these tests as is, but Simon said we should have them as tests instead of a notebook. We should have separate tests for `Model`, `View`, `Figure`, `Mesh`, `Line` etc.
But I won't write these until we are happy with the design.
The snippets here (to not call them tests) are just examples I was using for myself to check the behaviour, how everything could fit together, and whether it actually works.

Reply:
ok

Reply:
> So I think we should test all components in isolation and only have a few tests that make sure that they fit together.

I agree with your point about the `Figure`, but I don't feel that testing everything in isolation is sufficient? Shouldn't we test that we can fulfill the requirements, i.e., can create plots with certain features? "There is a model that can do X" is not a requirement.